### PR TITLE
Update add-domain

### PR DESCRIPTION
**Pull request by gafty**
## Domain/URL/IP(s) where you have found the Phishing:

bitelon.net

### Screenshot

<details><summary>Click to expand</summary>

</details>

### CODE REVIEW

**Comment by spirillen** [comment]
You need to fill out the form

**Comment by FMS2309** [comment]
Which form address please

**Comment by FMS2309** [comment]
My website is a false positive, I need it to be unblocked from the phishing database. What do I need to do? which form to fill?

**Review by FMS2309 — CHANGES_REQUESTED** [review]
My site has no virus is a false positive. By please remover the lock
www.wcei.com.br

File: add-domain [hunk 1]
```diff
@@ -1,3 +1,4 @@
+bitelon.net
```
Comment (FMS2309) [inline add-domain:1]: www.wcei.com.br

**Comment by FMS2309** [comment]
Domain/URL/IP(s) where you have found the Phishing:
www.wcei.com.br   (vírus total)

Impersonated domain:  www.wcei.com.br
Related external source: virus total
Describe the issue:   false positive
[https://www.virustotal.com/gui/url/cf8faa590e8b45d45338f241bc94886c31e899854770925812ce1f384e4923a2?nocache=1](url)

**Comment by spirillen** [comment]
image

**Pull request closed without merging**